archlinuxcn/python-torchgan-git/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# Maintainer: Butui Hu <hot123tea123@gmail.com>
+pkgname=python-torchgan-git
+pkgver=r99.9650398
+pkgrel=1
+pkgdesc="Research Framework for easy and efficient training of GANs based on PyTorch"
+arch=(any)
+url="https://github.com/torchgan/torchgan"
+license=('MIT')
+depends=(python-numpy python-pillow python-pytorch python-torchvision)
+makedepends=(git python-setuptools)
+provides=(python-torchgan)
```

Comment:
Usually `$pkgver` is included for a VCS package (e.g. `provides=("python-torchgan=$pkgver")`) so that other packages can depend on something like `python-torchgan>=0.0.2`.